src/main/java/be/seeseemelk/mockbukkit/MockBukkit.java

```diff
@@ -328,10 +328,22 @@ public static void unmock()
 	 * @return An instance of a mock plugin.
 	 */
 	public static MockPlugin createMockPlugin()
+	{
+		return createMockPlugin("MockPlugin");
+	}
+
+	/**
+	 * Creates a mock instance of a {@link JavaPlugin} implementation and gives you a chance to name the plugin. This plugin offers no functionality, but it does
+	 * allow a plugin that might enable and disable other plugins to be tested.
+	 *
+	 * @param pluginName A name of a new plugin.
+	 * @return An instance of a mock plugin.
+	 */
+	public static MockPlugin createMockPlugin(String pluginName)
```

Comment:
This sounds like a sensible addition, though some annotations can be added here (nitpicky I know)

```suggestion
	public static @NotNull MockPlugin createMockPlugin(@NotNull String pluginName)
```